albumentations/augmentations/transforms.py

```diff
@@ -459,6 +459,50 @@ def get_params(self):
     def apply_to_bbox(self, bbox, **params):
         return F.bbox_random_crop(bbox, self.height, self.width, **params)
 
+class RandomCenteredBBoxCrop(DualTransform):
```

Comment:
RandomCropNearBbox

Reply:
Is there is I need only to change the class name?

Reply:
yes